docs/index.html

```diff
@@ -0,0 +1,106 @@
+    
+</head>
+<body>
+
+    <H1>Lab 1 Walkthrough</H1>
+
+    <h2>Task 1: Adding to an Empty List</h2>
+
+    <div>
+        <a href="emptyListTask.html">Task 1 Before Link</a>
+        <p>This page shows the app with no tasks to do.</p>
+    </div>
+    <div>
+        <a href="Task1During.html">Task 1 During Link</a>
+        <p>This page shows the app while the user is in the process of typing out the name of the task that they are adding.
+        <br>
+        Note that the user types "Buy new John Grisham book" into the add new task bar.
+        </p>
+    </div>
+
+    <div>
+        <a href="Task1After.html">Task 1 After Link</a>
+        <p>This page shows the app after the item has been added to the to do list</p>
+    </div>
+
+    <h2>Task 2: Adding To a Non-Empty List</h2>
+
+    <div>
+        <a href="Task2Before.html">Task 2 Before Link</a>
+        <p>This page shows the app with one item in the list before adding another item</p>
+    </div>
+
+    <div>
+        <a href="Task2During.html">Task 2 Curing Link</a>
+        <p>This page shows the app while the user is in the process of typing out the name of the task that they are adding.
+            <br>
+            Note that the user types "Eat Lunch" into the add new task bar.
+            </p>
+    </div>
+
+    <div>
+        <a href="Task2After.html">Task 2 After Link</a>
+        <p>This page shows the app with all the desired items added</p>
+    </div>
+
+    <h2>Task 3: Marking Items Completed</h2>
+    
+    <div>
+        <a href="Task3Before.html">Task 3 Before Link</a>
+        <p>Note: In order to mark an item as completed, the user simply presses the button that the task is on</p>
+    </div>
+
+    <div>
+        <a href="Task3After.html">Task 3 After Link</a>
+        <p>This page shows the items that have been completed with the line through the text</p>
+    </div>
+
+    <h2>Task 4: Rename Items</h2>
+
+    <div>
+        <a href="Task4Before.html">Task 4 Before Link</a>
+        <p>Note: To rename an item click the edit icon and start typing in the button directly</p>
+    </div>
+
+    <div>
+        <a href="Task4After.html">Task 4 After Link</a>
+        <p>This page shows the renamed items in the list</p>
+    </div>
+
+    <h2>Task 5: Show Completed Items</h2>
+
+    <div>
+        <a href="Task5Before.html">Task 5 Before Link</a>
+        <p>
+        Note: To show only incompleted items, click the incoplete button on the top of the page to toggle between all and incomplete items.
+        </p>
+    </div>
+    
+    <div>
+        <a href="Task5 After.html">Task 5 After Link</a>
+        <p>This page shows the to do list without the completed items shown</p>
+    </div>
+
+    <h2>Task 6: Delete Completed Items</h2>
+
+        <div>
+            <a href="Task6Before.html">Task 6 Before Link</a>
+            <p>Note: To remove completed items, click the trash icon next to the submit new task button.
```

Comment:
Why would a trashcan button next to an add button signify to delete all completed items? Looks more like it's parallel to the Add button (do something to this row: add/delete).